webview/store/slices/keyValueTableDataSlice.js

```diff
@@ -72,6 +72,25 @@ const keyValueTableDataSlice = (set) => ({
       ),
     })),
 
+  handleFileUpload: (data, optionsType = COMMON.HEADERS, replaceValues) => {
+    const rows = []
+    for (let key in data) {
+      rows.push({
+        optionType: optionsType,
+        isChecked: true,
+        key,
+        value: data[key],
+        description: "",
+      })
+    }
+    set((state) => ({ 
+      keyValueTableData: [
+        ...(state.keyValueTableData.filter(v => data[v.key] === undefined && (!replaceValues || v.optionType !== optionsType))),
+        ...rows
+      ]
+    }))
+  },
+
```

Comment:
The code looks all good to me!
Thanks for your contribution, will add you to the contributor's list!